skiko/buildSrc/src/main/kotlin/tasks/configuration/JvmTasksConfiguration.kt

```diff
@@ -327,6 +327,9 @@ fun SkikoProjectContext.createLinkJvmBindings(
                 "-llog",
                 "-landroid",
                 "-latomic",
+                "-Wl",
+                "-z",
+                "max-page-size=16384",
```

Comment:
Documentation on the linked page https://developer.android.com/guide/practices/page-sizes#compile-r28 states that NDK r28+ always link with 16kb pages, do we still need to set these flags?

Reply:
You are right, the flags are not needed. I tested it again now and it seems I had some mix-up with the ndk version resolution order on my system